docs/ref/applications.txt

```diff
@@ -41,6 +41,22 @@ interact with installed applications, mainly for configuration and also for
 introspection. That's why the application registry maintains metadata in an
 :class:`~django.apps.AppConfig` instance for each installed application.
 
+Here is a comparison of "Project vs Application" for better understanding:
+
++--------------------+--------------------+------------------------------+
+| Topic              | Project            |Application                   |
++====================+====================+==============================+
+|Contains            | Settings           | Code, Views, Templates, ...  |
```

Comment:
The default project template also creates urls.py and wsgi.py. Some projects also store other things like templates and fixtures, see https://github.com/django/djangoproject.com/tree/master/djangoproject for an example.